test/helpers/fixture_deploy_helper.rb

```diff
@@ -62,7 +62,7 @@ def deploy_raw_fixtures(set, wait: true, bindings: {}, subset: nil)
 
   def deploy_dirs_without_profiling(dirs, wait: true, allow_protected_ns: false, prune: true, bindings: {},
     sha: "k#{SecureRandom.hex(6)}", kubectl_instance: nil, max_watch_seconds: nil, selector: nil,
-    protected_namespaces: nil, render_erb: true)
+    protected_namespaces: nil, render_erb: true, allow_globals: true)
```

Comment:
Is the plan to change the default when we remove `KubernetesDeploy`?

Reply:
yep.